setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 def get_long_description():
-    lines = open('README.rst', encoding='utf8').read().splitlines(False)
```

Comment:
I usually prefer to use ``io.open`` which is meant to be used as the equivalent of py3k ``open``.